lib/spack/spack/test/data/ninja/.gitignore

```diff
@@ -0,0 +1,2 @@
+.ninja_deps
+.ninja_log
```

Comment:
Running `ninja check` generates these files. Should I try to clean them up? How do I do that in pytest?